assets/dev/js/editor/elements/views/base.js

```diff
@@ -810,6 +814,23 @@ BaseElementView = BaseContainer.extend( {
 		return changed;
 	},
 
+	isCssIdControl( changedControl, bindingDynamicCssId ) {
+		return bindingDynamicCssId === changedControl;
+	},
+
+	updateCssId( dataBinding, change, settings, changedControl ) {
+		if ( ! change ) {
+			change = settings.attributes[ changedControl ];
+		}
+
+		if ( change && change.length ) {
+			dataBinding.el.closest( 'details' ).setAttribute( 'id', change );
```

Comment:
We can't hard code the `details` tag. It might be something else in other nested widgets.

Reply:
Nice catch!
TNX!!

Reply:
Done

Reply:
I still see the `details` :)

Reply:
Please check again
Sorry.